test/performance/specs/post-editor.spec.js

```diff
@@ -142,6 +143,7 @@ test.describe( 'Post Editor Performance', () => {
 
 		test( 'Setup the test post', async ( { admin, perfUtils, editor } ) => {
 			await admin.createNewPost();
+			await perfUtils.enterPostOnlyRenderingMode();
```

Comment:
Why would we need this one here? posts are "post-only" by default no?

Reply:
(Good catch by the way!)